defaults/config/modules/custom/the_build_utility/the_build_utility.info.yml

```diff
@@ -0,0 +1,10 @@
+name: the-build utility module
+description: 'Base settings from palantirnet/the-build.'
+type: module
+core: 8.x
+hide: true
+
+dependencies:
+  # Contrib modules
+  - config_split:config_split
```

Comment:
I'm not familiar with this `x:x` format. Is it something new?

Reply:
I was trying to follow the format recommended here: https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file - maybe this is dated?

Reply:
Nope, looks like [it's new-ish](https://www.drupal.org/node/2000204/revisions/view/9483535/9557149), but definitely not that new, so it's just me who is dated. :) Nice work.